docs/book/orders/cart-flow.rst

```diff
@@ -0,0 +1,48 @@
+.. index::
+   single: Cart flow
+
+Cart flow
+=========
+
```

Comment:
I suggest adding a bit of "storytelling" to the introduction. Like we've talked -> `A User comes to a Sylius shop and they say "Someone's been using my cart! And they filled it all up with some items!" 😄